### DISCUSSION TITLE
Add new Aldarin brewing vat to the Brewing plugin

### DISCUSSION THREAD

**Discussion by PaTcHiZzEl7397** [runelite/runelite#18460] [Ideas]
as the title says; The Varlamore pt 2 update added a new brewing vat on Aldarin, but hasn't yet been updated in the Brewing plugin

**Comment by Felanbird** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

https://github.com/InfernoStats/Brewing/issues/10
